code/modules/research/rdconsole.dm

```diff
@@ -833,24 +829,40 @@ won't update every console in existence) but it's more of a hassle to do. Also,
 				design_list["materials"] = materials_list
 				design_list["id"] = D.id
 				design_list["name"] = sanitize(D.name)
-				var/check_materials = 1
+				var/c = 1

```

Comment:
I'm actually curious what the point of this check_materials = 1 vs the c = 50 above @ line 776